### DISCUSSION TITLE
Reporting damages to municipalities via streetcomplete

### DISCUSSION THREAD

**Discussion by OpenGreenStreet** [streetcomplete/StreetComplete#3676] [Ideas]
I think it would be great if I could record damages via streetcomplete (for example broken pavement, broken street light etc.) by simply storing them as a note in StreetComplete. The municipalities could then evaluate these messages.

Reason: We mappers usually notice damage. Few municipalities do not have a reporting system.

**Comment by Dimitar5555** [comment]
That is a nice idea, but how are you planning to incorporate it in the app? Each municipality has a different reporting system (some of them don't have anything besides a phone number). Is this going to be some sort of a quest or something else?

**Comment by HolgerJeromin** [comment]
> by simply storing them as a note in StreetComplete

You can create a note at any place with touch-hold at any place. Perhaps this is what you miss already...

**Reply by OpenGreenStreet** [reply]
Yes, exactly: simply create a note (as a special category: "defect note"). This way, the local authorities have the location and the deficiency (preferably still with photo). The exciting question is, how does this become known and accessible to the municipalities?

**Reply by matkoniecz** [reply]
> You can create a note at any place with touch-hold at any place. Perhaps this is what you miss already...

This should be used only for issues with OSM map data. In case of widespread misuse it can be removed/changed to block misuses (or individual users can be blocked)

> The exciting question is, how does this become known and accessible to the municipalities?

The problem here is that it would require custom code for every place across world.

Some examples of complexity:

- things on privately owned areas may be out of scope for local government (and potholes/bad signs etc should be reported to operator of mall/restaurant/etc or owner of a house). Or not, depends on location.
- local government may accept solely reports via official documents (only on paper / accept also officially signed electronic documents - in case of Poland it would be https://epuap.gov.pl/ )
- email address (if available) to report issues would depend on location and type of issue, with classification of how issues are split would wildly differ across various places
- what would be worth reporting depends on location
- in many cases followup from reporter is needed
- things listed above will change over time (Kraków in Poland went from demanding submissions filed on paper, in person to allowing also ePUAP to starting to accept emails. For another aspect: there were also several reorganizations like creation of ZDMK, ZZM, shutting down ZIKIT and so on)

Based on my experience on reporting such things (over 1007 cases, including 494 fixed due to my intervention) I consider it as complex enough as deserving own separate project.

I would gladly contribute info for Kraków, though I am not expecting time to create such program/app (though it is on my TODO list from about 2015 to create something like that).

(please email me in case of such project starting or someone interested in funding its development, I am unwatching this thread as it is definitely not fitting StreetComplete which is an OSM editor - not a generic crowdsourcing tool)

**Reply by HolgerJeromin** [reply]
> This should be used only for issues with OSM map data.

Thanks for the comment!

**Reply by OpenGreenStreet** [reply]
Oh, I didn't think what could be attached to it.
My idea is just: with a special type of notes, things are documented. If a municipality/private person is interested in it, they can access it. If there's no interest, there's not.

**Comment by OpenGreenStreet** [comment]
If this is done by us mapers and the municipalities know about it, it could work from my point of view

**Comment by Cj-Malone** [comment]
Whether or not this is something that belongs in StreetComplete, I'd like to document [Open311](https://wiki.open311.org/GeoReport_v2/), it's a system to report maintenance issues like this.

(I think) The most popular implantation is [FixMyStreet](https://fixmystreet.org/) by mySociety.

They provide software to [local authorities](https://fix.bexley.gov.uk/around?zoom=4&lat=51.45683&lon=0.15341&js=1) for residents to report issues and for the authorities to respond to. It seems to be used in several countries, but on a local authority level, not country wide.

They also operate [FixMyStreet.com](https://osm.fixmystreet.com/around?lat=50.630098&lon=-1.172018&js=1) which integrates with the local authorities in the UK that use FixMyStreet and falls back to emailing other local authorities.

**Comment by mnalis** [comment]
Long pressing on the map allows you to choose between:
- open the note at that location (already mentioned by @HolgerJeromin)
  Problem with that is that I do not think it is realistic (at least in my country) to expect municipality to change the way they operate (i.e. to start looking at OSM notes with some hashtag instead [or in addition] to using whatever solution they are using at the moment). 
- open another app at that location, which allows for:
  - If your municipality have an geo-capable app, it should show in the list, and you can simply choose it and fill in extra data as required. 
  - If your municipality have an app, but it does not support `geo:`, you can ask them to add this functionality (and use functionality below in the meantime)
  - If your municipality does not have an app, but allow for some other way of reporting (like webpage, e-mail etc), you can still in StreetComplete choose `Share on OpenStreetMap`, and then copy/paste the link to OSM map containing the location to your municipality preferred method of communication.

**Reply by OpenGreenStreet** [reply]
Thanks for the feedback. This is exactly the problem: finding the right way to address the municipality. I would like to save that: no matter which municipality I am in: I see something leave a note and the municipality can use it if they are interested. Or the municipality is not interested: then it remains an unread note.

**Reply by HolgerJeromin** [reply]
> Or the municipality is not interested: then it remains an unread note.

...and the osm database will be filled with garbage as no one is bold enough to delete it.

**Reply by OpenGreenStreet** [reply]
There is of course something to it ... limit could be that perhaps about an expiration date (for example, a year)

**Reply by mnalis** [reply]
@OpenGreenStreet I agree with @HolgerJeromin here, it is a very bad idea unless you first get buy-in from municipality that they want to work that way. Because if you don't get buy-in from them, it's like leaving post-it notes on random bus station shelter expressing your wishes that public transport authority add a new bus route, and hoping that someone maybe will notice and that good will come out of it. It is highly unlikely... 

But even if you would get buy-in from them, it's still a bad idea. Purpose of OSM notes is to leave notes for other mappers **on how to improve OSM map**, and not for leaving notes for random people to do random stuff (or else people would start leaving notes like _"hey Joe, this is a cafe where I want us to have brunch tomorrow, please comment if that is OK for you"_. 

Because, even if the municipality would agree to be using OSM Notes, it would make it quite spammy for all the other OSM mappers, and so they would have to give up on using that system. 

@Cj-Malone mentioned `FixMyStreet`, and something like that is in my opinion best route to go - lobby your local municipality to implement such app so citizens can report problem. Or if they already have such an app, just use it. Not everything needs to be inside single app (much less in StreetComplete)

**Reply by OpenGreenStreet** [reply]
@mnalis, thank you for your feedback, which I can understand. I would have just found it nice to have both in one app. There doesn't seem to be a single city in Germany that participates in https://fixmystreet.org/sites/.
So it would be great to document very easily the tasks of the municipalities. (Of course, it would have to be a special category of notes that has nothing to do with the normal notes). I admit I have not looked at the legal issues involved.

**Reply by eginhard** [reply]
Cities won't automatically start checking OSM notes either. So if you have to lobby them anyway, it's better to suggest dedicated solutions for this purpose like FixMyStreet rather than abusing systems designed for something else.

**Reply by OpenGreenStreet** [reply]
Yes, I also see the challenges: Sure, municipalities won't automatically start checking OSM notes either: but if the info is publicly available, that creates pressure to act. (And SPAM could be prevented if only mappers with a certain number of solved tasks are allowed to create such special notes.)
